pythonwars/util.py

```diff
@@ -1,28 +1,32 @@
 #!/usr/bin/env python3
```

Comment:
I'm still not generally sold on having files named "util". can we move this class where it belongs (pythonwars.py or jsondict.py)?